source/includes/harvest/_scheduled_interviews.md

```diff
@@ -303,3 +326,235 @@ Retrieve an interview by its ID.
+	"location": "Big Conference Room"
+}
+```
+> The above returns a JSON response, structured like this:
+
+```json
+{
+  "id": 109170954,
+  "application_id": 102717457,
+  "external_event_id": "external_event_id_1",
+  "start": {
+    "date_time": "2018-12-12T13:15:00.000Z"
+  },
+  "end": {
+    "date_time": "2018-12-12T14:15:00.000Z"
+  },
+  "location": "Big Conference Room",
+  "status": "scheduled",
+  "created_at": "2018-10-17T19:22:07.302Z",
+  "updated_at": "2018-12-03T20:45:14.320Z",
+  "interview": {
+    "id": 8055374,
+    "name": "Executive Interview"
+  },
+  "organizer": {
+    "id": 102361,
+    "first_name": "Champ",
+    "last_name": "Telluride",
+    "name": "Champ Telluride",
+    "employee_id": null
+  },
+  "interviewers": [
+    {
+      "id": 102361,
+      "employee_id": null,
+      "name": "Champ Telluride",
+      "email": "champ.telluride.102361@example.com",
+      "response_status": "accepted",
+      "scorecard_id": null
+    },
+    {
+      "id": 46444,
+      "employee_id": "AAA1",
+      "name": "Lucius Barbarossa",
+      "email": "lucius.barbarossa.46444@example.com",
+      "response_status": "declined",
+      "scorecard_id": null
+    }
+  ]
+}
+```
+
+Create a new Scheduled Interview. 
+
+[See noteworthy response attributes.] (#the-scheduled-interview-object)
+
+### HTTP Request
+
+`POST https://harvest.greenhouse.io/v1/scheduled_interviews`
+
+### Headers
+
+Header | Description
+--------- | -----------
+On-Behalf-Of | ID of the user issuing this request. Will become the interview organizer. The user must have permission to manage the associcated application.
+
+### JSON Body Parameters
+
+Parameter | Required | Type | Description
+--------- | ----------- | ----------- | -----------
+application_id | Yes | integer | The id of the application for which we will scheduled an interview.
+interview_id | Yes | integer | The id of the interview we'd like to schedule. This id can be found in the [job stage object](#the-job-stage-object). It refers to an interview type (e.g. "Executive Interview" or "On-site Interview") that is associated with a hiring plan's interview step.  This differs from a "Scheduled Interview" which refers to an instantiated interview that is associated with an application, interviewers, start time, etc.
+interviewers[] | Yes | interviewer | Array of interviewers. Each must specify a user by user_id, email, or employee_id. Each must include a response status (one of needs_action, declined, tentative, or accepted). Each user must be a valid interviewer for the hiring plan.
+start | Yes | string | A datetime specifying when the interview starts. Must be provided in [ISO-8601](#general-considerations) format (e.g. 2018-11-05T13:12:14Z).
+end | Yes | string | A datetime specifying when the interview ends. Must be provided in [ISO-8601](#general-considerations) format (e.g. 2018-11-05T13:12:14Z).
+external_event_id | Yes | string | A unique identifer for this interview.
```

Comment:
Are these required? I kinda thought they were optional for some reason...

Reply:
`external_event_id` is currently required - should that be changed?

Reply:
I'm fine to leave it as-is for now. It might be a little annoying if the third party doesn't use external IDs, but I don't know how common that would be.